### PR TITLE
fix: unable to create user if Accounts_ManuallyApproveNewUsers is enabled

### PR DESCRIPTION
**Pull request by debdutdeb**
## Further comments

Discussion - https://open.rocket.chat/channel/support?msg=Y5WMDNcW9tYM82oab, https://open.rocket.chat/channel/support?msg=zvyWcCbW55fJtFpYr

### CODE REVIEW

**Comment by sampaiodiego** [comment]
thx Deb.. I guess this should land in a patch release for 6.2, right?

**Comment by debdutdeb** [comment]
> thx Deb.. I guess this should land in a patch release for 6.2, right?

Ah idk really .. last I knew only security fixes were in patch 🙈 I'll make it patch then.

File: .changeset/heavy-years-repeat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': minor
```
Comment (debdutdeb) [inline .changeset/heavy-years-repeat.md:2]:
```suggestion
'@rocket.chat/meteor': patch
```

**Comment by debdutdeb** [comment]
@sampaiodiego if patch, then this should target release-6.2.0 branch right?

**Comment by sampaiodiego** [comment]
> @sampaiodiego if patch, then this should target release-6.2.0 branch right?

actually only security fixes are back ported to old (but still supported) releases.. 😉

File: apps/meteor/app/authentication/server/startup/index.js [hunk 2]
```diff
@@ -196,7 +196,7 @@ const onCreateUserAsync = async function (options, user = {}) {
 	if (!user.active) {
 		const destinations = [];
 		const usersInRole = await Roles.findUsersInRole('admin');
-		await usersInRole.toArray().forEach((adminUser) => {
+		(await usersInRole.toArray()).forEach((adminUser) => {
```
Comment (sampaiodiego) [inline apps/meteor/app/authentication/server/startup/index.js:199]: this is actually what they tried to do =)
```suggestion
		await usersInRole.forEach((adminUser) => {
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Pull request merged**